.ui/pgCategoryDetail.pgx

```diff
@@ -0,0 +1,64 @@
+{
+	"components": [
+		{
+			"className": ".sf-page",
+			"id": "7dd3-eb16-f5e0-3035",
+			"initialized": true,
+			"props": {
+				"children": [
+					"86bf-5dec-4deb-0ed6",
+					"319e-211d-a87b-8d37",
+					"9c8a-99c0-5d01-7907"
+				],
+				"name": "pgCategoryDetail",
+				"orientation": "PORTRAIT",
+				"parent": null
+			},
+			"type": "Page",
+			"userProps": {},
+			"version": "0.0.1.a"
+		},
+		{
+			"className": ".sf-statusBar",
+			"id": "86bf-5dec-4deb-0ed6",
+			"props": {
+				"children": [],
+				"isRemovable": false,
+				"name": "statusBar",
+				"parent": "7dd3-eb16-f5e0-3035"
+			},
+			"type": "StatusBar",
+			"userProps": {}
+		},
+		{
+			"className": ".sf-headerBar",
+			"id": "319e-211d-a87b-8d37",
+			"props": {
+				"children": [],
+				"isRemovable": false,
+				"name": "headerBar",
+				"parent": "7dd3-eb16-f5e0-3035"
+			},
+			"type": "HeaderBar",
+			"userProps": {
+				"backgroundColor": "rgba( 255, 255, 255, 1 )",
+				"visible": true
```

Comment:
we shouldn't be using user props inside page

Reply:
done.

Reply:
done.